### PR TITLE
doc: add news for 2.4.1

### PR DESCRIPTION
**Pull request by HashidaTKS**

### CODE REVIEW

**Comment by HashidaTKS** [comment]
@kou @komainu8 

Woul you review this when you have time?

File: news/index.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ upper_level: ../
...
+
+  * Added support for PostgreSQL 15.
+  
+  * Dropped support for PostgreSQL 10.
+
+    Because PostgreSQL 10 will reach EOL on November 2022.
+
+  * [[`` &? `` operator for jsonb type][query-jsonb-v2]] Added translation about how to perform full text search with indexes against jsonb type values. 
+    [GitHub:pgroonga/pgroonga.github.io#83][Patched by askdkc]
+
```
Comment (komainu8) [inline news/index.md:20]: We need to add `### Thanks`.

**Review by komainu8 — CHANGES_REQUESTED** [review]
Could you confirm my comments?

**Comment by HashidaTKS** [comment]
@komainu8 

Thanks, I have added a "Thanks" section.

**Pull request merged**